habitat_baselines/train_ppo.py

```diff
@@ -111,8 +111,8 @@ def construct_envs(args):
     baseline_configs = []
 
     basic_config = cfg_env(config_paths=args.task_config, opts=args.opts)
-
-    scenes = PointNavDatasetV1.get_scenes_to_load(basic_config.DATASET)
+    dataset = make_dataset(basic_config.DATASET.TYPE)
+    scenes = dataset.__class__.get_scenes_to_load(basic_config.DATASET)
```

Comment:
Is this the right way to do it? It should be a class method.

Reply:
@apsdehal, what exactly do you mean?

Reply:
Class and static methods are also callable/work as expected when using an instance of the class.
i.e. `scenes = dataset.get_scenes_to_load(basic_config.DATASET)`

Reply:
Fixed.